### DISCUSSION TITLE
【图片占位符】希望能对图片尚未加载出来时显示个图片占位标识

### DISCUSSION THREAD

**Discussion by txuw** [tangly1024/NotionNext#2015] [Ideas]
**为什么提出这个新的特性改动**
当读者阅读博客时，快速浏览时，在当前版本，图片无法加载出来会以空的形式展示，而读者此时并不知道此处有个图片，容易影响文章的阅读体验

**描述一下你推荐的解决方案**
希望在图片尚未加载时，提供一个图片占位标识

**描述一下你考虑过的其它替代解决方案**
1. 以一个loading的加载块去告诉读者此处的图片尚未加载出来？

**补充说明**
能展示出此处有个图片就可以很好的优化阅读体验了

**Comment by tangly1024** [comment]
新版本中，图片未加载完成时，会显示成一个色块，相比原先做了优化。
